sql/function/list-broken-indexes/jsonb/locked.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE logs (
+  id int,
+  record jsonb
+);
+
+INSERT INTO logs VALUES (1, '{"message": "Hello World"}');
+INSERT INTO logs VALUES (2, '{"message": "This is a pen"}');
+INSERT INTO logs VALUES (3, '{"message": "Good-by World"}');
+CREATE INDEX pgroonga_index ON logs
+  USING pgroonga (record pgroonga_jsonb_ops_v2);
+
+select pgroonga_command(
+  'lock_acquire',
+  ARRAY['target_name', replace(pgroonga_table_name('pgroonga_index'), 'Sources', 'JSONPaths') || '_0']
```

Comment:
```sql
select replace(pgroonga_table_name('pgroonga_index'), 'Sources', 'JSONPaths') || '_0';
     ?column?
-------------------
 JSONPaths203941_0
(1 row)
```

What is a better way to get table names for jsonb?

Reply:
We can use this because we don't have shorter way than this.